projects/ngx-charts-on-fhir/src/lib/data-layer-browser/data-layer-browser.component.html

```diff
@@ -4,11 +4,23 @@ <h2>Available Data Layers</h2>
     <mat-form-field class="filter" appearance="outline">
       <mat-icon matPrefix fontIcon="search"></mat-icon>
       <input matInput [formControl]="filterControl" placeholder="Search" />
-      <button mat-icon-button matSuffix (click)="filterControl.setValue('')"><mat-icon fontIcon="close"></mat-icon></button>
+      <button mat-icon-button matSuffix (click)="filterControl.setValue('')"><mat-icon
+          fontIcon="close"></mat-icon></button>
     </mat-form-field>
   </header>
+  <section class="progress-bar-section">
+    <mat-progress-bar *ngIf="isProgress"
+        class="progress-bar-margin"
+        mode="indeterminate"
+        [value]="value">
```

Comment:
why are we passing a value to indeterminate progress bar?

Reply:
I am remove the value.